deploy/.env.example

```diff
@@ -0,0 +1,42 @@
+# COMMON
+LOG_LEVEL=INFO
+NETWORK=mainnet
+ENABLE_HEALTH_SERVER='True'
+HEALTH_SERVER_PORT='8080'
```

Comment:
I suggest we replace single quotes with a double as they're more common in env files.